Formula/akamai.rb

```diff
@@ -0,0 +1,27 @@
+class Akamai < Formula
+  desc "CLI toolkit for working with Akamai's API from the command-line."
+  homepage "https://github.com/akamai/cli"
+  url "https://github.com/akamai/cli.git",
+    :tag => "0.1.0"
```

Comment:
This would need `:revision =>` too, but instead please try using
```
  url "https://github.com/akamai/cli/archive/0.1.0.tar.gz"
  sha256 "68a49db1c397ff55185b06252d1f2590395c6193566cdb52b2c910dd7d6c3889"
```